packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -0,0 +1,428 @@
+								__(
+									'Are you sure you want to delete this thread?'
+								)
+							}
+							confirmAction={ () => {
+								onCommentDelete( thread.id );
+								setActionState( false );
+							} }
+							discardAction={ () => setActionState( false ) }
+						/>
+					) }
+			</>
+		);
+	};
+
+	return (
+		<>
+			{
+				// If there are no threads, show a message indicating no threads are available.
+				( ! Array.isArray( threads ) || threads.length === 0 ) && (
+					<VStack
+						alignment="left"
+						className="editor-collab-sidebar-panel__thread"
+						justify="flex-start"
+						spacing="3"
+					>
+						{
+							// translators: message displayed when there are no comments available
+							__( 'No comments available' )
+						}
+					</VStack>
+				)
+			}
+
+			{ Array.isArray( threads ) &&
+				threads.length > 0 &&
+				threads.map( ( thread ) => (
+					<VStack
+						key={ thread.id }
+						className={ clsx(
+							'editor-collab-sidebar-panel__thread',
+							{
+								'editor-collab-sidebar-panel__active-thread':
+									blockCommentId &&
+									blockCommentId === thread.id,
+							}
+						) }
+						id={ thread.id }
+						spacing="3"
+					>
+						<CommentBoard thread={ thread } />
+						{ 'reply' === actionState?.action &&
+							thread.id === actionState?.id && (
+								<VStack
+									alignment="left"
+									spacing="3"
+									justify="flex-start"
+									className="editor-collab-sidebar-panel__comment-field"
+								>
+									<CommentForm
+										onSubmit={ ( inputComment ) => {
+											onAddReply(
+												inputComment,
+												thread.id
+											);
+											setActionState( false );
+										} }
+										onCancel={ () =>
+											setActionState( false )
+										}
+									/>
+								</VStack>
+							) }
+						{ 0 < thread?.reply?.length &&
+							thread.reply.map( ( reply ) => (
+								<VStack
+									key={ reply.id }
+									className="editor-collab-sidebar-panel__child-thread"
+									id={ reply.id }
+									spacing="2"
+								>
+									<CommentBoard
+										thread={ reply }
+										parentThread={ thread }
+									/>
+								</VStack>
+							) ) }
+					</VStack>
+				) ) }
+		</>
+	);
+}
+
+/**
+ * EditComment component.
+ *
+ * @param {Object}   props          - The component props.
+ * @param {Function} props.onSubmit - The function to call when updating the comment.
+ * @param {Function} props.onCancel - The function to call when canceling the comment update.
+ * @param {Object}   props.thread   - The comment thread object.
+ * @return {JSX.Element} The CommentForm component.
+ */
+function CommentForm( { onSubmit, onCancel, thread } ) {
+	const [ inputComment, setInputComment ] = useState(
+		thread?.content?.raw ?? ''
+	);
+
+	return (
+		<>
+			<TextareaControl
+				__nextHasNoMarginBottom
+				className="editor-collab-sidebar-panel__comment-field-textarea"
```

Comment:
Some of those classnames might still be unused and potentially removable. Let's make an audit across the PR and remove all unnecessary classnames. This one seems to be a candidate for removal.